packages/clay-badge/src/ClayBadge.soy

```diff
@@ -10,7 +10,7 @@
 
 	{let $classes kind="text"}
 		badge badge-pill
-		{sp}badge-{$style ?: 'default'}
+		{sp}badge-{$style}
```

Comment:
If default value is `primary` as defined in the STATE this should be `{sp}badge-{$style ?: 'primary'}`